### PR TITLE
Don't exit early if a test failed

### PR DESCRIPTION
**Pull request by wesleybalvanz-wf**
@seangerhardt-wf I think this will prevent the early exit so that you will get the xml reports.

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: goverge/coverage.py [hunk 1]
```diff
@@ -20,14 +20,18 @@
...
 def check_failed(return_code):
     """Check if the test run failed and exit with the correct code if it did.
     :type return_code: int
     :param return_code: The return code of a test run
     """
     if return_code != 0:
-        os._exit(return_code)
+        global _failed
+        _failed = True
+        print("This is FAILED's value: {}".format(_failed))
```
Comment (aldenpeterson-wf) [inline goverge/coverage.py]: do you still need this?

Comment (wesleybalvanz-wf) [reply goverge/coverage.py]: Nope debugging statement that I clearly forgot to remove....

File: goverge/test/test_coverage.py [hunk 2]
```diff
@@ -21,14 +21,15 @@
...
+from goverge import coverage
 
 
 class TestCheckFailed(unittest.TestCase):
-
     @patch('goverge.coverage.os._exit')
     def test_check_failed(self, mock_exit):
         check_failed(1)
-        mock_exit.assert_called_with(1)
+        print(coverage._failed)
```
Comment (aldenpeterson-wf) [inline goverge/test/test_coverage.py]: another one... :)

Comment (wesleybalvanz-wf) [reply goverge/test/test_coverage.py]: (facepalm)

**Comment by aldenpeterson-wf** [comment]
lgtm

**Comment by seangerhardt-wf** [comment]
+1
@wesleybalvanz-wf I ran this with failing unit tests and the xunit reports appeared as expected.

**Pull request merged**